numpyro/contrib/distributions/continuous.py

```diff
@@ -120,9 +252,56 @@ def variance(self):
         return np.broadcast_to(self.scale ** 2, self.batch_shape)
 
 
+class LogNormal(TransformedDistribution):
+    arg_constraints = {'loc': constraints.real, 'scale': constraints.positive}
+    support = constraints.positive
+    reparametrized_params = [(0, 'loc'), (1, 'scale')]
```

Comment:
just curious: why we need `0`, `1` here?

Reply:
I'm not too happy with this either. Will change it to something else, if there is a better way to do this. `0, 1` denote the position of the args, and this makes it easy to test. Another way might be to have an OrderedDict denoting whether the parameter is reparametrized `{'p1': False, 'p2': True}`, but that seems equally kludgey. Will think about it.

Reply:
Okay, changed to list. I'm just using `inspect` in the test now, so this kludge doesn't need to get into the actual code.